# Tools-Resources

# Bioinformatics Tools and Public Resources

This repository lists commonly used tools, databases, and public resources relevant for clinical and research genomics analyses, particularly WGS, WGBS, and variant annotation workflows.

## Analysis Tools (Software)

- **R** – Statistical computing and scripting  
  [https://www.r-project.org](https://www.r-project.org)

- **Python** – General-purpose programming, data analysis  
  [https://www.python.org](https://www.python.org)

- **Nextflow** – Workflow management system for reproducible pipelines  
  [https://www.nextflow.io](https://www.nextflow.io)

- **BWA-MEM2** – Alignment of sequencing reads to reference genomes  
  [https://github.com/bwa-mem2/bwa-mem2](https://github.com/bwa-mem2/bwa-mem2)

- **GATK** – Variant calling and processing  
  [https://gatk.broadinstitute.org](https://gatk.broadinstitute.org)

- **VEP (Variant Effect Predictor)** – Variant annotation  
  [https://www.ensembl.org/info/docs/tools/vep/index.html](https://www.ensembl.org/info/docs/tools/vep/index.html)

---

## Public Genomic Databases

- **1000 Genomes Project** – Population variant frequencies  
  [https://www.internationalgenome.org](https://www.internationalgenome.org)

- **gnomAD** – Population variant frequencies and allele counts  
  [https://gnomad.broadinstitute.org](https://gnomad.broadinstitute.org)

- **ClinVar** – Clinical variant interpretations  
  [https://www.ncbi.nlm.nih.gov/clinvar/](https://www.ncbi.nlm.nih.gov/clinvar/)

- **Ensembl** – Gene, transcript, and genome annotation  
  [https://www.ensembl.org](https://www.ensembl.org)

- **UCSC Genome Browser** – Genome visualization and annotations  
  [https://genome.ucsc.edu](https://genome.ucsc.edu)

---

## Notes

These resources are commonly used for:

- Annotation, filtering, and prioritization of variants  
- Reference genome and gene-level information  
- Reproducible genomic data analyses  
- Integrating workflows in clinical genomics pipelines
